pyof/foundation/base.py

```diff
@@ -115,6 +115,12 @@ def __xor__(self, other):
     def __rxor__(self, other):
         return self.value ^ other
 
+    def __lshift__(self, n):
+        return self.value << n
```

Comment:
Hum... may be 'n' is better, since you shift the value 'n positions' (or n bits). I think that, in this specific case, 'n' is more suitable than 'other'. Does it make sense?